rep-0199.rst

```diff
@@ -0,0 +1,407 @@
+
+The following shows an example frame hierarchy for a single serial manipulator with an EEF model attached to ``flange`` and one application-specific tool frame (``eef_tcp``)::
+
+  base_link
+  ├ base
+  └ link_1
+    └ ..
+      └ link_N
+        └ flange
+          ├ tool0
+          ├ eef_base_link
+          │   └ ..
+          │     └ eef_link_N
+          └ eef_tcp
+
+Note the '``eef_``' prefix on the links in the EEF sub-hierarchy to prevent name clashes with the main robot model.
+
+Note also that ``eef_tcp`` is a child of ``flange`` and not of ``eef_base_link``.
+This is in accordance with `Application-Specific Tool Frames`_, as the EEF's TCP (in this example) is the result of a three-point calibration which was performed relative to the flange of the robot.
+
+Multi-group (asymmetric)
+------------------------
+
+An example frame hierarchy for a setup that consists of two groups, a 6 axis industrial manipulator and a 2 axis positioner (or turntable).
+
+Both are placed in the same work cell and share a common ``world`` frame::
+
+  world
+  ├ ..
+  ├ robot_base_link
+  │ ├ robot_base
+  │ └ robot_link_1
+  │   └ ..
+  │     └ robot_link_N
+  │       └ robot_flange
+  │         └ robot_tool0
+  └ positioner_base_link
+    ├ positioner_base
+    └ positioner_link_1
+      └ positioner_link_2
+        └ positioner_flange
+          └ positioner_tool0
+
+Note the '``robot_``' and '``positioner_``' prefixes on all frames.
+
+Multi-group (symmetric)
+-----------------------
+
+The following shows an example frame hierarchy for a dual-arm robot that consists of two identical manipulators that are mirrored around a shared base.
+Each arm sub-hierarchy has been given a prefix corresponding to its relative position::
+
+  base_link
+  ├ base
+  ├ left_base_link
+  │ ├ left_base
+  │ └ left_link_1
+  │   └ ..
+  │     └ left_link_N
+  │       └ left_flange
+  │         └ left_tool0
+  └ right_base_link
+    ├ right_base
+    └ right_link_1
+      └ ..
+        └ right_link_N
+          └ right_flange
+            └ right_tool0
+
+Note that ``base_link`` in this example is the root of the entire robot structure and should be used when integrating the robot into a larger assembly.
+
+
+Vendor Nomenclature Mapping
+===========================
+
+This section shows a mapping between vendor-specific frame nomenclature and the ``base`` and ``tool0`` frames as defined by this REP.
+
+Note that for most vendors, ``tool0`` corresponds to an *all-zeros* tool frame configuration as described in the `tool0`_ section.
+The names listed here in the *Vendor Name* column refer to the generic names for frames as used in the documentation of the control systems of the respective vendors, and not necessarily to any specific configurations of those frames.
+
++------------------+------------+---------------+
+|                  | This REP   | Vendor Name   |
++==================+============+===============+
+| ABB              | ``base``   | Base          |
```

Comment:
I'd keep this list out of the REP and put it into the wiki. The wiki page is easier to maintain for future manufacurers and changes in vendor-specific implementations. Here a link to the wiki page would suffice.

Reply:
>  The wiki page is easier to maintain for future manufacurers and changes in vendor-specific implementations

While I agree that a wiki would be easier to edit, the number of changes to this table will probably be very few. Industrial robot OEMs are not likely to change the coordinate frames they use in their controllers (because of legacy, documentation, conventions, training, etc), so this table is only expected to grow, not change.

However, I'm not entirely sold on having this table in the REP either, so I'll count your comment as a :-1: and wait for some more feedback.